.gitignore

```diff
@@ -7,4 +7,9 @@ artifacts
 vendor
 .DS_Store
 .*env
-dist
\ No newline at end of file
+dist
+
+/go.work
+/go.work.sum
```

Comment:
The go.work and sum files are required to be able to run e2e test from the sub module test within the repo.

Reply:
Yes, I removed these lines.